Editor/Resources/EditorWindow/Pages/GameLiftPluginBucketPopup.uxml

```diff
@@ -2,20 +2,23 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 <ui:UXML xmlns:ui="UnityEngine.UIElements">
-    <Style src="../common.uss"/>
-    <ui:VisualElement>
-        <ui:Label name="UserProfilePageBootstrapPopupTitle" text="You might incur charges for creating an S3 Bucket..."/>
-        <ui:Label name="UserProfilePageBootstrapPopupDescription" text="Check and see if you have exceeded the AWS Free Tier usage &#10;limits or your period expired."/>
-        <ui:TextField name="UserProfilePageBootstrapPopupBucketText" label="Bucket Name" value="gamelift-bootstrap-012345678912-eu-west-1"/>
-        <ui:VisualElement class="separator separator--horizontal">
+    <Style src="./GameLiftPluginBucketPopup.uss"/>
+    <ui:VisualElement class="separator separator--large">
+        <ui:VisualElement class="separator">
+            <ui:Label name="UserProfilePageBootstrapPopupTitle" class="popup__heading" text="You might incur charges for creating an S3 Bucket..."/>
+            <ui:Label name="UserProfilePageBootstrapPopupDescription" text="Check and see if you have exceeded the AWS Free Tier usage &#10;limits or your period expired."/>
             <ui:VisualElement name="UserProfilePageBootstrapPopupFreeTierLinkParent" class="external-link">
                 <ui:Label name="UserProfilePageBootstrapPopupFreeTierLink"/>
                 <ui:Image name="ExternalLinkIcon" class="icon--small"/>
             </ui:VisualElement>
-            <ui:VisualElement class="separator separator--horizontal float-right">
-                <ui:Button name="UserProfilePageBootstrapPopupCancelButton" text="Cancel"/>
-                <ui:Button name="UserProfilePageBootstrapPopupContinueButton" text="Continue"/>
+            <ui:VisualElement class="separator separator--large separator--horizontal separator--centered">
+                <ui:Label name="UserProfilePageBootstrapPopupBucketLabel"/>
+                <ui:TextField name="UserProfilePageBootstrapPopupBucketText" class="text-input" value="gamelift-bootstrap-012345678912-eu-west-1"/>
```

Comment:
The mock indicates this has a brighter color, `#e4e4e4` and this seems to match other inputs in the mocks. Please add that color variable and add it to text-input in `common.uss`